androdeb

```diff
@@ -221,9 +224,15 @@ if [ ! -z "$KERNELHDRS" ]; then
 fi
 
 # Package kernel headers
-if [ ! -z "$KERNELSRC" ]; then
-	echo "Building and updating kernel headers from kernel source dir ($KERNELSRC)"
-	$spath/bcc/build-kheaders-targz.sh ${KERNELSRC} $TDIR_ABS/kh.tgz > /dev/null
+if [ ! -z "$KERNELSRC" -o ! -z "$KERNELOUT" ]; then
```

Comment:
This part may not work. If src is provided but out isn't provided, then it would still end up calling the build headers script?

Reply:
Here's what I tested:

- Provided just `--kernelsrc`: Works as before - no change if provided just this option from an in-tree build. If pointed at the source for an out-of-tree build, errors out since `autogenerated.h` is not present.
- Provided just `--kernelout`: Packaged up just the headers in the kernel output tree for an out-of-tree build. Verified by inspection of the tar contents (as it only contained the generated headers and wouldn't actually be useful).
- Provided both `--kernelsrc` and `--kernelout`: Worked as expected for an out-of-tree kernel build. Successfully built and ran bcc/criticalstat on a target device with an out-of-tree kernel build.